kubernetes/ansible/roles/sunbird-monitoring/templates/prometheus-operator.yaml

```diff
@@ -83,6 +83,9 @@ alertmanager:
       group_interval: 5m
       repeat_interval: 4h
       routes:
+      - match_re:
+         alertname: "{{ ignore_alerts_list }}"
```

Comment:
Use templating to change it as per your requirement.
```
"{{ '^(' + (ignore_alerts_list | join('|') + ')' }}"
```

Reply:
define this variable in defaults/main.yaml as 
```
igonore_alert_list_custom: []
```
So that if anyone wants to override custom alerts they can use this variable.

Comment this variable, on how to use that.